internal/x509fork/cert_pool.go

```diff
@@ -234,9 +234,7 @@ func (s *CertPool) AppendCertsFromPEM(pemCerts []byte) (ok bool) {
 
 // Subjects returns a list of the DER-encoded subjects of
 // all of the certificates in the pool.
-//
-// Deprecated: if s was returned by [SystemCertPool], Subjects
-// will not include the system roots.
+// Undeprecated in this fork.
```

Comment:
It would be weird to have only "Undeprecated in this fork" after merging this PR. Can we add a link to the original package?

https://pkg.go.dev/crypto/x509#CertPool.Subjects